Twig/Extension/ThemeExtension.php

```diff
@@ -45,9 +45,9 @@ public function getFunctions()
     /**
      * Displays Wordpress theme header (and administration menu bar if available).
      */
```

Comment:
Thank you, could you also update PHPdoc on methods your updated please? Just to add the new argument:

```
      * @param string|null $name
```